gifs_test.go

```diff
@@ -1,15 +1,14 @@
-package gifs_go_test
+package gifs
```

Comment:
Could we keep gifs_test because we need the tests to be runnable and use the imports like users would in their code?